account_invoice_kanban/README.rst

```diff
@@ -0,0 +1,91 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Account Invoice Kanban
+=======================
+
+
+Introduction
```

Comment:
Remove this header too please - introduction is assumed to be the text immediately below the main header